front/lib/api/assistant/global_agents.ts

```diff
@@ -17,9 +18,8 @@ import {
   MISTRAL_MEDIUM_MODEL_CONFIG,
   MISTRAL_SMALL_MODEL_CONFIG,
 } from "@dust-tt/types";
-import { AgentConfigurationType, GlobalAgentStatus } from "@dust-tt/types";
-import { PlanType } from "@dust-tt/types";
-import { DustAPI } from "@dust-tt/types";
+import {} from "@dust-tt/types";
```

Comment:
✂️ ?

Reply:
Not sure how these keep coming back. Will delete